mne/surface.py

```diff
@@ -1853,19 +1853,94 @@ def _vtk_smooth(pd, smooth):
     return out
 
 
-def _warn_missing_chs(info, dig_image, after_warp, verbose=None):
-    """Warn that channels are missing."""
-    # ensure that each electrode contact was marked in at least one voxel
-    missing = set(np.arange(1, len(info.ch_names) + 1)).difference(
-        set(np.unique(np.array(dig_image.dataobj))))
-    missing_ch = [info.ch_names[idx - 1] for idx in missing]
-    if missing_ch and verbose != 'error':
-        warn(f'Channel{_pl(missing_ch)} '
-             f'{", ".join(repr(ch) for ch in missing_ch)} not assigned '
-             'voxels ' +
-             (f' after applying {after_warp}' if after_warp else ''))
+@verbose
+def warp_montage(montage, moving, static, reg_affine, sdr_morph, verbose=None):
+    """Warp a montage to a template with image volumes using SDR.
+
+    .. note:: This is likely only applicable for channels inside the brain
+              (intracranial electrodes).
+
+    Parameters
+    ----------
+    montage : instance of mne.channels.DigMontage
+        The montage object containing the channels.
+    %(moving)s
+    %(static)s
+    %(reg_affine)s
+    %(sdr_morph)s
+    %(verbose)s
+
+    Returns
+    -------
+    montage_warped : mne.channels.DigMontage
+        The modified montage object containing the channels.
+    """
+    _require_version('nibabel', 'SDR morph', '2.1.0')
+    _require_version('dipy', 'SDR morph', '1.6.0')
```

Comment:
It's a specific type so let's be more specific
```suggestion
    _require_version('dipy', 'warping points using SDR', '1.6.0')
```